test/integration/StreamrClient.test.js

```diff
@@ -210,7 +210,7 @@ describe('StreamrClient Connection', () => {
                 ])
                 done()
             })
-        })
+        }, 300000)
```

Comment:
This is quite a long timeout. We can probably declare the test failing in e.g. 30 seconds or 1 minute instead of 5 minutes.